DOC-12531-Feedback on set flush_param

supritha-kumar opened the pull request:
Changed "The cbepctl flush_param exp_pager_stime command sets the time interval for disk cleanup " to

"The cbepctl flush_param exp_pager_stime command sets the time interval to scan for expired items, and erase them from memory and disk."

https://jira.issues.couchbase.com/browse/DOC-12531

supritha-kumar committed:
DOC-12531-Feedback on set flush_param

Changed "The cbepctl flush_param exp_pager_stime command sets the time interval for disk cleanup " to

"The cbepctl flush_param exp_pager_stime command sets the time interval to scan for expired items, and erase them from memory and disk."

https://jira.issues.couchbase.com/browse/DOC-12531

supritha-kumar committed:
Update set-flush_param.adoc

updated disk cleanup to asynchronous expiry

Pallavi-Janardhan commented on modules/cli/pages/cbepctl/set-flush_param.adoc:
```diff
@@ -70,7 +70,7 @@ Note also that if the scanner runs at the same time that index updates are being
 
 exp_pager_stime::
-The `cbepctl flush_param exp_pager_stime` command sets the time interval for disk cleanup.
+The `cbepctl flush_param exp_pager_stime` command sets the time interval to scan for expired items, and erase them from memory and disk.
```
We can remove the comma after items and before and erase.

Pallavi-Janardhan reviewed (approved):
LGTM

supritha-kumar committed:
Updated set-flush_param.adoc

removed comma from "scan for expired items and erase them from memory and disk"
